frontend/app/routes/_public+/apply+/$id+/federal-provincial-territorial-benefits.tsx

```diff
@@ -163,8 +164,8 @@ export default function AccessToDentalInsuranceQuestion() {
         )}
       </section>
       <div className="flex flex-wrap items-center gap-3">
-        <ButtonLink id="back-button" to="/apply">
-          <FontAwesomeIcon icon={faChevronLeft} className="me-3 block size-4" />
+        <ButtonLink to={`/apply/${id}/dental-insurance`}>
+          <FontAwesomeIcon icon={faChevronLeft} className="ms-3 block size-4" />
```

Comment:
className `ms-3` must be `me-3` otherwise the margin is not set properly. I need to read more about `margin-inline-*`. I think we would need to use `margin` left or right for our button icons.

With `ms-3`
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/128839d6-e3d3-47ba-b14b-bf0a9e8a692b)

With `me-3`
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/ae2f4bb0-8046-49d2-bcc9-ce7c8a89c600)